### DISCUSSION TITLE
Enable module maintainers to subscribe to / monitor merge confidence

### DISCUSSION THREAD

**Discussion by voxpelli** [renovatebot/renovate#29163] [Suggest an Idea]
### Tell us more.

When launching a new version of a module a worry is often: Will it break the ecosystem?

The merge confidence is great for users of the module, but there's no way for the releaser of the module to monitor it and/or get notified of a bad such confidence.

I would love to work with you to enable such a subscription for packages I (co-)maintain, like `mocha`, `npm-run-all2` etc

---

One idea would be to enable one to sign up to your site, another to expose some kind of RSS-feed or similar, a third to add some kind of report url (similar to the web [Reporting API](https://www.w3.org/TR/reporting-1/)) to a package's `package.json` and have renovate send data to it (in a way that can't be spoofed – signed or through trusted IP:s or such)

**Comment by jamietanna** [comment]
(Just dropping in to say that yes, this is still something we're interested in and want to do!)
